src/code.sh

```
@@ -341,14 +341,24 @@ main() {
        mkdir subjob_output
 
+       # wait 60 seconds before trying to download all files as some files
+       # can still be in an open state due to parallel uploading still
+       # commencing even after subjobs have completed successful
+       echo "Waiting 60 seconds to ensure all files are hopefully in a closed state"
+       sleep 30
+
        for fusion in "${fusion_lists[@]}"; do
-	       echo $fusion
 	       echo ${fusion%.*}
 	       tar_name=$prefix_$fusion.tar
```

Shellcheck (0.10.0)

[warning] prefix_ is referenced but not assigned (did you mean 'prefix'?).

(SC2154)